rds/rds_provider.go

```diff
@@ -78,36 +89,32 @@ func (r *RDS) CreateDatabase(db *crd.Database) (string, error) {
 	// search for the instance
 	log.Printf("Trying to find db instance %v\n", db.Spec.DBName)
 	k := &rds.DescribeDBInstancesInput{DBInstanceIdentifier: input.DBInstanceIdentifier}
-	res := r.rdsclient().DescribeDBInstancesRequest(k)
-	_, err = res.Send(context.Background())
-	if err != nil && err.Error() != rds.ErrCodeDBInstanceNotFoundFault {
+
+	_, err = r.rdsclient().DescribeDBInstances(ctx, k)
+	if err != nil && err.Error() != new(rdstypes.DBInstanceNotFoundFault).Error() {
```

Comment:
why do you need to new it?

Reply:
removed the new()

Reply:
`rdstypes.DBInstanceNotFoundFault` error receiver is a pointer. so before call on any method on this defined error it's require to create the object. if we  call `new(rdstypes.DBInstanceNotFoundFault)` , it returns a pointer to newly created  error of type `rdstypes.DBInstanceNotFoundFault` and on that pointer we can call `Error() method.

If you remove the `new()` it's a syntax error.

Reply:
Added the new() back in